test/testcafe/spec/IdentifyRegistration_spec.js

```diff
@@ -128,7 +129,7 @@ test.requestHooks(mock)('should show errors after empty required fields are focu
   await t.expect(registrationPage.hasEmailErrorMessage()).eql(true);
 });
 
-test.requestHooks(enrollProfileErrorMock)('should show email field validation errors', async t => {
+test.meta('v3', false).requestHooks(enrollProfileErrorMock)('should show email field validation errors', async t => {
```

Comment:
Every other assertion in this test works except for these: 

```
await t.expect(registrationPage.getNthEmailErrorMessage(0)).eql('\'Email\' must be in the form of an email address');
  await t.expect(registrationPage.getNthEmailErrorMessage(1)).eql('Provided value for property \'Email\' does not match required pattern');
```

The reason for this is because in v3, we only display 1 field level error message, whereas v2 displays x number of field level errors. 

I reached out to @alexdahl-okta regarding this to determine how we want to handle it in v3. Will update once we have the go forward approach.

Reply:
after speaking with Alex, we are enabling multiple field level messages to be displayed. I have implemented this and enabled this test for parity. See `FieldErrorContainer` component.